### ISSUE TITLE
DB Scripts

### ISSUE THREAD

**Issue by Bomber12321** [ikismail/ShoppingCart#51]
It would be nice if you can send me the db scripts for oracle.
email : aniloamit123@gmail.com
thanks

**Comment by sairathikanti** [comment]
yes please could you send DBScripts email : vamshi.sai.9866@gmail.com

**Comment by Haneesh001** [comment]
Can u please send me the database file in email haneeshalapati007@gmail.com

**Comment by khuong16** [comment]
Can u please send me the database file in email tranminhkhuongph08470@gmail.com

**Comment by ikismail** [comment]
Duplicate Issue - https://github.com/ikismail/ShoppingCart/issues/22#issuecomment-380333291
